components/infobox/commons/infobox_person.lua

```diff
@@ -463,4 +469,19 @@ function Person._createAgeCalculationErrorMessage(text)
 	end
 end
 
+function Person:_flipNameOrder(args)
+	if not Logic.readBool(args.nonameflip) and Table.includes(
+		{
+		'China',
+		'Taiwan',
+		'Hong Kong',
+		'Vietnam',
+		'South Korea',
+		'Cambodia'
+		}, args.country) then
```

Comment:
maybe add a comment mentioning that args.country is already standardized/cleaned before getting passed to this function